### PR TITLE
Cpi

### PR DESCRIPTION
**Pull request by gursharan-info**

### CODE REVIEW

File: libs/sharepoint/bipp/sharepoint/sharepoint.cfg [hunk 1]
```diff
@@ -0,0 +1,6 @@
+SHAREPOINT_HOSTNAME=isbhydmoh-my.sharepoint.com
+MAIN_FOLDER=Documents/India Data Portal (IDP)
+INDIAPULSE_FOLDER=IDP_Gramener/IndiaPulse
+IDP_FOLDER=idp_datasets
```
Comment (Suryakandukuri) [inline libs/sharepoint/bipp/sharepoint/sharepoint.cfg]: Please add this file to gitignore

File: projects/cpi/data/external/State_LGD_CPI_Inflation.csv [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+Andaman And Nicobar Islands,35
+All India,0
+Andhra Pradesh,28
+Arunachal Pradesh,12
+Assam,18
+Bihar,10
+Chandigarh,4
+Chhattisgarh,22
+Dadra And Nagar Haveli,26
+Daman And Diu,25
```
Comment (Suryakandukuri) [inline projects/cpi/data/external/State_LGD_CPI_Inflation.csv:11]: was this staged by choice?

**Review by Suryakandukuri — APPROVED** [review]

File: projects/cpi/data/external/State_LGD_CPI_Inflation.csv (hunk 2 above)
Comment (Suryakandukuri) [reply projects/cpi/data/external/State_LGD_CPI_Inflation.csv:11]: Please check and respond

**Review by Suryakandukuri — CHANGES_REQUESTED** [review]

File: projects/cpi/data/external/State_LGD_CPI_Inflation.csv (hunk 2 above)
Comment (gursharan-info) [reply projects/cpi/data/external/State_LGD_CPI_Inflation.csv:11]: Yes...!!  This is the LGD mapping file with some specific spellings for this dataset

File: libs/sharepoint/bipp/sharepoint/sharepoint.cfg (hunk 1 above)
Comment (gursharan-info) [reply libs/sharepoint/bipp/sharepoint/sharepoint.cfg]: I don't how this came into the folder. I never created or used a cfg file. Will remove and add exclusion for any .cfg file in gitignore

**Pull request merged**